### PR TITLE
Design atervinningsstation

### PR DESCRIPTION
**Pull request by NoahAkkad**
Add styling to the Atervinningsstation.css page.
Remove the old styling from the Återvinningsstationpage.js.

### CODE REVIEW

File: src/pages/InfoPages/Atervinningsstation.css [hunk 1]
```diff
@@ -0,0 +1,322 @@
+.station-station-directory {
```
Comment (Abbe91) [inline src/pages/InfoPages/Atervinningsstation.css:1]: Why is the name station-station? Can we make it a station-directory?

File: src/pages/InfoPages/Atervinningsstation.css [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+  background-color: #f9f9f9;
+  border: 1px solid #ddd;
+  border-radius: 8px;
+  box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  width: 100%;
+}
+
+.station-station-directory p {
```
Comment (Abbe91) [inline src/pages/InfoPages/Atervinningsstation.css:10]: the same for the name

File: src/pages/InfoPages/Atervinningsstation.css [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+.station-station-directory a {
+  text-decoration: none;
+  color: #333;
+  font-size: 0.8rem;
+  margin-right: 0.5rem;
+  border-bottom: 1px solid #333;
+}
+
+.station-station-directory a:last-child {
```
Comment (Abbe91) [inline src/pages/InfoPages/Atervinningsstation.css:23]: please update the name in all files.

File: src/pages/InfoPages/Atervinningsstation.css [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+}
+
+.station-services-description-btn:hover {
+  background-color: #d4b036;
+  transform: scale(1.05);
+}
+
+@media (max-width: 600px) {
+  .station-center-section h3 {
+      font-size: 1.25rem;
```
Comment (Abbe91) [inline src/pages/InfoPages/Atervinningsstation.css:320]: is it necessary to have a different font size just for h3?

File: src/pages/InfoPages/Atervinningsstation.js [hunk 5]
```diff
@@ -97,65 +99,47 @@ const Atervinningsstation = () => {
...
                 href="https://goteborg.se/wps/portal/start/bygga-bo-och-leva-hallbart/avfall-och-atervinning/har-lamnar-hushall-avfall/atervinningsstationer/hitta-atervinningsstationer"
                 key={i}
-                style={{
-                  fontSize: "0.8rem",
-                  marginRight: "0.5rem",
-                  borderBottom: "1px solid black",
-                }}
+                // this anchor tag is not have title attribute
+                // i will add title attribute
+                // when you approve the changes i will remove this comment
```
Comment (Abbe91) [inline src/pages/InfoPages/Atervinningsstation.js:112]: Can you explain this? I did not get your point on this.

Comment (NoahAkkad) [reply src/pages/InfoPages/Atervinningsstation.js:112]: done

File: src/pages/InfoPages/Atervinningsstation.css (hunk 1 above)
Comment (NoahAkkad) [reply src/pages/InfoPages/Atervinningsstation.css:1]: done

File: src/pages/InfoPages/Atervinningsstation.css (hunk 3 above)
Comment (NoahAkkad) [reply src/pages/InfoPages/Atervinningsstation.css:23]: done

File: src/pages/InfoPages/Atervinningsstation.css (hunk 4 above)
Comment (NoahAkkad) [reply src/pages/InfoPages/Atervinningsstation.css:320]: Yes, it necessary to reduce the font of the tag H3 when placing the mobile phone, as in the previous design of the Atervinningscentral page.
It's more beautiful

File: src/pages/InfoPages/Atervinningsstation.css (hunk 2 above)
Comment (NoahAkkad) [reply src/pages/InfoPages/Atervinningsstation.css:10]: done

**Pull request merged**